terraform/aws/packer.json

```diff
@@ -5,7 +5,7 @@
     "source_ami": "ami-80861296",
     "instance_type": "t2.medium",
     "ssh_username": "ubuntu",
-    "ami_name": "hashistack {{timestamp}}",
+    "ami_name": "cgbaker hashistack-nvidia {{timestamp}}",
```

Comment:
I would remove `cgbaker` from ami name.  Identifying it as nvidia or tensorrt demo ami is sufficient i think.

Reply:
👍